examples/c/i2c_example.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Author: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
```

Comment:
I guess the author is you, so please put your name here.
Also modify the following description.